docs/package/gluon-scheduled-domain-switch.rst

```diff
@@ -0,0 +1,30 @@
+gluon-scheduled-domain-switch
+=============================
+
+This package allows to switch a routers domain at a given point
+in time. This is needed e.g. for communities switching mesh-protocols
+(IBSS / 802.11s) without using both simultaneously.
+
+Nodes will switch if this defined *switch-time* has passed. In case the node was
+powered off when this was supposed to happen, it might not be able to set it's clock
```

Comment:
while this was supposed to happen, it might not be able to get aquire the correct time.